AirportTicketBookingExercise/App/Utils/CsvActionsHelper.cs

```diff
@@ -0,0 +1,67 @@
+﻿using ATB.Data.Extensions;
+using ATB.Data.Models;
+using CsvHelper;
+using CsvHelper.Configuration;
+using System.Globalization;
+using System.Xml.Schema;
+
+namespace AirportTicketBookingExercise.App.Utils
+{
+    public class CsvActionsHelper
```

Comment:
this is not an App related helper, should be in logic. I don't UI cares about this